example/serving/components/pubsub.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
```

Comment:
default file created by, its not needed for running samples locally

Reply:
Given that the `example-http-binding` needs to be there for the output binding example to work it's better to be explicit here and not have these files magically show up for the user